sdk/src/types/block/address/bech32.rs

```diff
@@ -172,11 +176,15 @@ impl Bech32Address {
 
 impl core::fmt::Display for Bech32Address {
     fn fmt(&self, f: &mut core::fmt::Formatter<'_>) -> core::fmt::Result {
-        write!(
-            f,
-            "{}",
-            bech32::encode::<bech32::Bech32>(self.hrp.0, &self.inner.pack_to_vec(),).unwrap()
-        )
+        let bytes = if self.inner.is_multi() {
+            core::iter::once(MultiAddress::KIND)
+                .chain(Blake2b256::digest(self.inner.pack_to_vec()))
+                .collect()
+        } else {
+            self.inner.pack_to_vec()
+        };
+
+        write!(f, "{}", bech32::encode::<bech32::Bech32>(self.hrp.0, &bytes).unwrap())
```

Comment:
We never documented this unwrap btw. we don't care here?

Reply:
documented it